pages/common/matlab.md

```diff
@@ -0,0 +1,16 @@
+# matlab
+
+> Numerical computation environment by MathWorks.
+> More information: <https://uk.mathworks.com/help/matlab/>.
+
+- Run without splash screen during startup:
+
+`matlab -nosplash`
```

Comment:
Missing dash? Not a matlab user, so I'm just checking to make sure.

Reply:
Yep that's correct, MATLAB only uses one dash even in linux (it's a weird one, I know...)

Reply:
Oh, that's odd!